Docker/hooks/pre_build

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [[ "$DOCKERFILE_PATH" == *-ARM ]];
+then
```

Comment:
nitpicky: you usually write this on the same line with if and you don't need to quote vars when using test.

Reply:
@Alkarex PR on my end incoming soon.